tests/Omnipay/Stripe/Message/PurchaseRequestTest.php

```diff
@@ -6,18 +6,61 @@
 
 class PurchaseRequestTest extends TestCase
 {
+    /** @var PurchaseRequest $request */
+    private $request;
+    private $card;
+
     public function setUp()
     {
         $this->request = new PurchaseRequest($this->getHttpClient(), $this->getHttpRequest());
+        $this->card = $this->getValidCard();
         $this->request->initialize(
             array(
                 'amount' => '10.00',
                 'currency' => 'USD',
-                'card' => $this->getValidCard(),
+                'card' => $this->card,
+                'statementDescriptor' => "FOO"
             )
         );
     }
 
+    public function testShouldReturnDataVersionOfCard()
```

Comment:
Ah, I intended to write "returnData" then I changed to return arrayVersionOfCard. mixed up the sentence